### ISSUE TITLE
Support to free decentralized social network

### ISSUE THREAD

**Issue by airon90** [Pirate-Parties-International/PPI-party-info#6]
Is it possible to add another variable (e.g. "decentrfederSN") in order to add a reference to the decentralized and federated social networks such as Diaspora, Friendica, ecc.

Many Pirate Parties created an account on these "piratish" social network. AFAIK here is the list:
- [x] PPIT:  partitopirata@friendica.ambientedigital.org 
- [x] PPES Galiciia: piratasdegalicia@poddery.com 
- [x]  PPUY: partidopiratauruguay@diasp.eu
- [x] PPNO: piratpartiet_norge@joindiaspora.com
- [x] PPIN: piratepin@diasp.org
- [x] PPAU: piratepartyau@diasp.org 
- [x] PPNL Leuven:  ppleuven@diasp.eu
- [x] PPDE Bonn:  piratenbonn@pod.geraspora.de
- [x] PPBR:  piratas_br@diasporabr.com.br 
- [x] PPFR:  partipirate@joindiaspora.com 
- [ ] PPDE Bochum:  piratenpartei_bochum@pod.geraspora.de
- [x] PPDE:  piratenpartei@joindiaspora.com
- [ ] PPDE Gifhorn:  piratengifhorn@pod.geraspora.de 
- [ ] PPDE Hamburg:  piratenhamburg@pirati.ca 
- [ ] PPNL Limburg:  pplimburg@diasp.eu 
- [ ] PPNL Niedersachsen: piratennds@pod.geraspora.de 
- [ ] PPDE VG:  piratenparteivg@diasp.org 
- [x] PPCA:  piratepartyca@diasp.org 
- [x] PPGR:  piratepartygreece@joindiaspora.com 
- [x] PPBA: piratskapartija@joindiaspora.com
- [ ] PPUS Washington:  wa_pirate_party@diasp.org 
- [ ] PP DE Hessen:  piratenpartei_hessen@joindiaspora.com
- [ ] AG Europa: ag_europa@pirati.ca
- [ ] PPDE RTK: piratenpartei_rtk@despora.de 

Note: These aren't email address but account IDs: e.g. [partitopirata@friendica.ambientedigital.org](http://friendica.ambientedigital.org/u/partitopirata)

**Comment by Putr** [comment]
The JSON (and the API code) was designed in a way that adding new variables to the socialNetworks will not break functionality. When a new social network is added it is present in the data, returned by the API but not shown on the front-end.

If you are a first to add a new social network also update the relevant documentation (when I finally write it) because the value for a social network can either be a string (just an ID) or an object of values (like for twitter - id and hashtags)

To add diaspora just do:

```
"socialNetworks": { 
....
    "diaspora": "diasporaID",
....
}
```
Than submit an issue so that that social network is implemented on the frontend.

PS: Sorry for leaving this project hanging for 2 years. I've been doing some work on this for a while, but it's slow going. But we finally managed to organize a hackaton scheduled for this Friday that should do some major cleanup/upgrades so that this project can FINALLY be launched.

**Comment by Putr** [comment]
@Airon90 Any chance you could add this data to the dataset? The documentation is done as well as contribute info. Just check the README.md.

**Comment by Airon90** [comment]
I'll do that tomorrow ;)

**Comment by Airon90** [comment]
By now I added all national accounts and left all regional and working team accounts.

**Comment by Putr** [comment]
I've merged your merge requests. When you're done just close the issue.

PS: If you do any more merges, please refrence the issue ID in the commit message (see my commits). Thanks!

**Comment by Airon90** [comment]
Ok, thank you very much and sorry for not referencing the merges :)

**Comment by Putr** [comment]
Have all been added? Can we close this issue?

**Comment by Airon90** [comment]
No, I have to create regional parties and then add the missing data. During these days I'll do this work

**Comment by collentine** [comment]
@airon90 has this been fixed?

**Comment by airon90** [comment]
TBH I don't remember what I did and now I'm not a member anymore. I suggest you to check every data as long as maybe some channels are unused.

**Comment by PeterTheOne** [comment]
decentralized social network are part of the data_specification.md, but not all from the list above are part of the dataset. so this should stay open.
